File: CommonPassion_Backend/Data/ApiModels/ApiStanding.cs
using System;
namespace CommonPassion_Backend.Data.ApiModels
{
    public class ApiStanding
    {
        public string get { get; set; }
        public ParametersStanding parameters { get; set; }
        public object[] errors { get; set; }
        public int results { get; set; }
        public Paging paging { get; set; }
        public ResponseStanding[] response { get; set; }
    }

    public class ParametersStanding
    {
        public string league { get; set; }
        public string season { get; set; }
    }


    public class ResponseStanding
    {
        public LeagueWithStanding league { get; set; }
    }


    public class Standing
    {
        public int rank { get; set; }
        public Team team { get; set; }
        public int points { get; set; }
        public int goalsDiff { get; set; }
        public string group { get; set; }
        public string form { get; set; }
        public string status { get; set; }
        public string description { get; set; }
        public All all { get; set; }
        public Home home { get; set; }
        public Away away { get; set; }
        public DateTime update { get; set; }
    }

   

    public class All
    {
        public int played { get; set; }
        public int win { get; set; }
        public int draw { get; set; }
        public int lose { get; set; }
        public GoalsTotal goals { get; set; }
    }

    public class GoalsTotal
    {
        public int _for { get; set; }
        public int against { get; set; }
    }

    public class Home
    {
        public int played { get; set; }
        public int win { get; set; }
        public int draw { get; set; }
        public int lose { get; set; }
        public GoalsHome goals { get; set; }
    }

    public class GoalsHome
    {
        public int _for { get; set; }
        public int against { get; set; }
    }

    public class Away
    {
        public int played { get; set; }
        public int win { get; set; }
        public int draw { get; set; }
        public int lose { get; set; }
        public GoalsAway goals { get; set; }
    }

    public class GoalsAway
    {
        public int _for { get; set; }
        public int? against { get; set; }
    }

    }

File: CommonPassion_Frontend/CommonPassion/src/app/models/api_standing.ts
import { Paging } from "./RootLeague";
import {League} from './RootLeague';
import { Team } from "./api_team";


export interface ApiStanding {
    get: string;
    parameters: ParametersStanding;
    errors: any[];
    results: number;
    paging: Paging;
    response: ResponseStanding[];
}

export interface ParametersStanding {
    league: string;
    season: string;
}


export interface ResponseStanding {
    league: League;
}


export interface Standing {
    rank: number;
    team: Team;
    points: number;
    goalsDiff: number;
    group: string;
    form: string;
    status: string;
    description: string;
    all: All;
    home: Home;
    away: Away;
    update: Date | string;
}



export interface All {
    played: number;
    win: number;
    draw: number;
    lose: number;
    goals: GoalsTotal;
}

export interface GoalsTotal {
    _for: number;
    against: number;
}

export interface Home {
    played: number;
    win: number;
    draw: number;
    lose: number;
    goals: GoalsHome;
}

export interface GoalsHome {
    _for: number;
    against: number;
}

export interface Away {
    played: number;
    win: number;
    draw: number;
    lose: number;
    goals: GoalsAway;
}

export interface GoalsAway {
    _for: number;
    against: number ;
}